### ISSUE TITLE
[Export] Mapping multiple classification store fields into one column does not work

### ISSUE THREAD

**Issue by vanchelosa** [w-vision/DataDefinitions#293]
![Снимок экрана от 2021-02-02 11-36-20](https://user-images.githubusercontent.com/74260768/106573636-f22db880-654a-11eb-94d1-ffef4446acd0.png)

If you map several fields of the classification store into one column of the export file (csv, xml, json, it does not matter), then only one value from the first exported object is exported, for all other objects in this column it will be empty. This works fine for normal fields, but it doesn't work for classification store fields.

**Comment by dpfaffenbauer** [comment]
cause you named it the same way. try renaming it.

**Comment by vanchelosa** [comment]
Yes, I named them the same on purpose. As you can see in the screenshot above, several groups use the same attribute, in the export file I want to get the values of this attribute in one column. Yes, I named them the same on purpose. As you can see in the screenshot above, several groups use the same attribute, in the export file I want to get the values of this attribute in one column.

**Comment by dpfaffenbauer** [comment]
What happens if you rename it? that is the problem here.

**Comment by vanchelosa** [comment]
I have many classification store groups, about three hundred, many use the same attribute. If I make several fields in one column, then I get a file with thousands of columns, if I use a unique column name for the same attribute in each group, I get a file with many thousands of columns.

**Comment by dpfaffenbauer** [comment]
well. Then it is just unsupported and not a bug.

**Comment by vanchelosa** [comment]
Got it, I just checked it on other fields and it works with them and I thought that this is a bug with the fields of the classification store.
